src/categories/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+
+class Categories(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    name = models.CharField(max_length=50)
+    parent_id = models.ForeignKey("self", related_name="categories", on_delete=models.SET_NULL, blank=True, null=True)
```

Comment:
`related_name` là gì?

Reply:
I just edit

Reply:
I just ask, why you remove it?

Reply:
Sorry.. related_name  la tên cho  quan hệ 2 table

Reply:
ở đây mỗi 1 table mà?

Reply:
Dạ vậy suy ra không cần thiết related_name ạ! Để e xóa